controllers/machinehealthcheck_controller_test.go

```diff
@@ -717,94 +717,98 @@ func TestMachineHealthCheck_Reconcile(t *testing.T) {
 		}).Should(Equal(1))
 	})
 
-	t.Run("when in a MachineSet, unhealthy machines should be deleted", func(t *testing.T) {
```

Comment:
@jan-est would you be able to reorder so this newly added test cases are after the existing ones? It's currently showing up as a huge diff and I think these are just new test cases? If you can do that, it should make this easier to review. Thanks!

Reply:
@ncdc I reorder the test cases. Hope it helped.